<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditClass</class>
 <widget class="QDialog" name="DialogEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение информации о классе</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>124</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="classNameLabel">
        <property name="text">
         <string>Название класса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="classNameLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="classTeacherLabel">
        <property name="text">
         <string>ФИО классного руководителя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="classTeacherLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="teachersPhoneLabel">
        <property name="text">
         <string>Телефон классного руководителя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="teachersPhoneLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="applyClassButton">
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelClassButton">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
